runbotics/runbotics-desktop/src/action/sharepoint/excel/sharepoint-excel.action-handler.ts

```diff
@@ -41,22 +41,26 @@ export default class SharepointExcelActionHandler extends StatelessActionHandler
             throw new Error(SharePointExcelErrorMessage.getCellIncorrectInput());
         }
 
-        const { value, text, numberFormat } = await this.excelService.getCell({
+        const cellValue = await this.excelService.getCell({
             column: column.toString(),
             row: row.toString()
         });
 
-        if (numberFormat.includes('@') || numberFormat.includes('%')) {
-            return text;
-        }
-
-        return value;
+        return cellValue;
     }
 
     async getRange(
         input: SharepointTypes.SharepointExcelGetRangeActionInput
-    ): Promise<SharepointTypes.SharepointExcelGetRangeActionOutput> {
-        return this.excelService.getRange(input.range);
+    ): Promise<ExcelCellValue[][]> {
+        const properRange = input.range.match(/^([a-zA-Z]+\$?\d+):(\$?[a-zA-Z]+\$?\d+)/);
```

Comment:
i think it's been already used in the project. Add this to RegexPattern enum as e.g. EXCEL_CELLS_RANGE